bedrock/security/management/commands/update_security_advisories.py

```diff
@@ -31,6 +32,8 @@
 
 SM_RE = re.compile('seamonkey', flags=re.IGNORECASE)
 FNULL = open(os.devnull, 'w')
+HOF_FILES = ['client.yml', 'web.yml']
+HOF_DIRECTORY = 'bug-bounty-hof'
```

Comment:
Would it be useful to have this (these?) in `settings`?

Reply:
We could, but they won't change without a change to the repo and I think coordinating a push would be fine. Also I don't expect them to change.

Reply:
Cool. Carry on.